data.sql

```diff
@@ -68,7 +68,7 @@ UPDATE animals SET owner_id = 'Dean Winchester' WHERE name = 'Boarmon';
 
```

Comment:
- [ ] It would be better if the file changes matches up with the project. This project is specifically about the Animals table but I could see Vets, Owner, etc which is not part of the project. Kindly follow the project guidelines and delete all code that are not related to the project